bin/run_celery_worker_log_parser

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+curr_dir=$( dirname "${BASH_SOURCE[0]}" )
+cd $( dirname $curr_dir)
+source /etc/profile.d/treeherder.sh
+
+if [[ ${NEW_RELIC_LICENSE_KEY+isset} = isset ]]; then
+    NEWRELIC_ADMIN="newrelic-admin run-program"
+fi
+
+if [ -f ../venv/bin/celery ]; then
+    source ../venv/bin/activate
+    CELERY=../venv/bin/celery
+else
+    CELERY=celery
+fi
+
```

Comment:
There's a lot of duplication between these different celery scripts, it would be nice to consolidate some of the logic into a common file at some point.